R/entry.R

```diff
@@ -0,0 +1,78 @@
+LanguageGlossaryEntry <- R6::R6Class("LanguageGlossaryEntry",
+  private = list(
+    .lang = NA_character_,
+    .term = NA_character_,
+    .defn = NA_character_
+  ),
+  public = list(
+    initialize = function(term, defn, lang = "en") {
+      stopifnot(rlang::is_scalar_character(term))
+      stopifnot(rlang::is_scalar_character(defn))
+      stopifnot(rlang::is_scalar_character(lang))
+      private$.term <- term
+      private$.defn <- defn
+      private$.lang <- lang
+    },
+    language =  function() {
+      private$.lang
+    },
+    term = function() {
+      private$.term
+    },
+    definition = function() {
+      private$.defn
+    },
+    print = function() {
```

Comment:
I'm new to R6 - is it usual to provide a way to convert to string as well?

Reply:
I'm not sure what you mean here...